orca-core/src/main/java/com/netflix/spinnaker/orca/pipeline/StageDefinitionBuilder.java

```diff
@@ -42,7 +42,7 @@
 import javax.annotation.Nonnull;
 import javax.annotation.Nullable;
 
-public interface StageDefinitionBuilder {
+public interface StageDefinitionBuilder extends SimpleStageDefinitionBuilder {
```

Comment:
Is there now a bunch of duplication w/ `SimpleStageDefinitionBuilder`?

Reply:
I'd also like to get an answer on this as well. What's the purpose of `SimpleStageDefinitionBuilder` if this one extends it and remains unchanged?